File: 0217-contains-duplicate/0217-contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        Map<Integer,Integer> frequencyMap = new HashMap<>();
        for (int num : nums) {
            frequencyMap.put(num, frequencyMap.getOrDefault(num, 0) + 1);
            if(frequencyMap.getOrDefault(num,0)>1)return true;
        }
        return false;
    }
}

File: 0217-contains-duplicate/0217-contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    nums.sort();
    let x= nums[0];
    for(let i=1;i<nums.length;i++){
        if(nums[i]==x)return true;
        x=nums[i];
    }
    return false;
};